content/blog/migrating-vsphere-virtual-volumes-to-kubevirt-with-hpe-csi-driver-for-kubernetes.md

```diff
@@ -0,0 +1,432 @@
+      dev: "{{ source_host_device }}"
+      fstype: ext4
+      opts: "-L {{ filesystem_label }}"
+
+  - name: Mount filesystem and persist
+    mount:
+      path: "{{ var_lib_mysql }}"
+      state: mounted
+      src: "LABEL={{ filesystem_label }}"
+      fstype: ext4
+
+# Install MariaDB on both VMs and download Test_DB
+- hosts: common
+  become: yes
+  tasks:
+  - name: Install prereqs manually
+    command: dnf install -y virt-what mariadb-server
+
+  - name: Download Test_DB
+    unarchive:
+      src: "{{ test_db_tgz }}"
+      remote_src: yes
+      dest: "{{ ansible_user_dir }}"
+      creates: test_db
+
+# Disable/stop MariaDB on destination and remove mountpoint
+- hosts: kubevirt
+  become: yes
+  tasks:
+  - name: Disable MariaDB
+    service:
+      enabled: no
+      state: stopped
+      name: mariadb
+  - name: Remove default MariaDB datadir
+    file: 
+      path: "{{ var_lib_mysql }}"
+      state: directory
+      recurse: true
+
+# Start/enable MariaDB and deploy TestDB on source
+- hosts: legacy
+  become: yes
+  tasks:
+  - name: Set permissions
+    file:
+      path: "{{ var_lib_mysql }}"
+      owner: mysql
+      group: mysql
+      mode: "0755"
+      seuser: system_u
+      serole: object_r
+      setype: mysqld_db_t
+      selevel: s0
+
+  - name: Enable MariaDB
+    service:
+      name: mariadb
+      state: started
+      enabled: true
+
+  - name: Deploy Test_DB
+    shell: mysql < employees_partitioned.sql
+    args:
+      creates: "{{ var_lib_mysql }}/employees"
+      chdir: "{{ ansible_user_dir }}/test_db"
+
+  - name: Test Test_DB
+    shell: mysql -t < test_employees_sha.sql && mysql -t <<< 'select @@hostname as 'VM';'
+    args:
+      chdir: "{{ ansible_user_dir }}/test_db"
+    register: test_db_results
+
+  - name: Show results
+    pause:
+      prompt: "{{ test_db_results.stdout }}"
+      seconds: 10
+```
+
+Another detail in the playbook that might need clarification is that the Test_DB sources is downloaded to the destination VM only because the database queries are needed for validation of the cloning and migration.
+
+The last few output lines of the playbook reveal that the database was deployed and tested properly.
+
+```shell
+...
++---------+--------+
+| summary | result |
++---------+--------+
+| CRC     | OK     |
+| count   | OK     |
++---------+--------+
++----------------+
+| VM             |
++----------------+
+| mariadb-legacy |
++----------------+:
+ok: [mariadb-legacy]
+
+PLAY RECAP *************************************************************************
+mariadb-kubevirt           : ok=6    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
+mariadb-legacy             : ok=15   changed=2    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
+```
+
+Once the prerequisites have been installed, we can also inspect the hypervisor platform each VM is running on, to ensure what we’re doing is actually being executed where we expect it.
```

Comment:
Change to "Once the prerequisites have been installed, **you** can also inspect the hypervisor platform each VM is running on to ensure what **you're** doing is actually being e**xecuted where expected**."